canvas_modules/common-canvas/locales/common-properties/locales/en.json

```diff
@@ -58,8 +58,10 @@
   "expression.fields.title": "Fields",
   "expression.values.title": "Values",
   "expression.field.column": "Field",
+  "expression.field.column.description": "",
   "expression.storage.column": "Storage",
   "expression.value.column": "Value",
+  "expression.value.column.description": "",
```

Comment:
@matthoward366 @tomlyn For this issue, I want to display the description in the info icon tooltip only if it exists. I have code to handle that in the expression table. Is having empty strings here okay? Omitting it will result in "Missing message" errors. 

References: https://github.com/formatjs/formatjs/pull/488 and https://github.com/formatjs/formatjs/issues/465